paper-fab.html

```diff
@@ -109,7 +109,8 @@
       }
     </style>
 
-    <iron-icon id="icon" src="[[src]]" icon="[[icon]]"></iron-icon>
+    <iron-icon id="icon" hidden$="{{!_computeIsIconFab(icon)}}" src="[[src]]" icon="[[icon]]"></iron-icon>
+    <span id="fab-text" hidden$="{{_computeIsIconFab(icon)}}">{{label}}</span>
```

Comment:
How about adding a style for this so that it doesn't overflow? Something like
```
    width: 100%;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: clip;
```